src/operations/replace_one.ts

```diff
@@ -19,15 +19,16 @@ export interface ReplaceOptions extends CommandOperationOptions {
   /** When true, creates a new document if no document matches the query */
   upsert?: boolean;
 
-  // FIXME:
+  // FIXME: ????!!!! neal why?
```

Comment:
😿

Reply:
Maybe the intention is to have a `replace` operation that does both, or a `replace_multi`?

Reply:
🤔

Reply:
Git blame says I left the FIXME there thats why I left 'neal why?' lol oops! looks like the option doesn't need to be there, it isn't a replaceOne option but it is an update option, maybe during typing this interface was being shared.